addons/website_sale_slides/models/product_product.py

```diff
@@ -9,6 +9,15 @@ class Product(models.Model):
 
     channel_ids = fields.One2many('slide.channel', 'product_id', string='Courses')
 
+    def _is_add_to_cart_allowed(self):
+        # Allow course related products to the cart regardless of product's rules
+        self.ensure_one()
+        res = super()._is_add_to_cart_allowed()
+        return res or self.env['slide.channel'].sudo().search_count([
```

Comment:
Actually, looking at it again, I think it's better to return `res or bool( [your code] )`, to ensure the return type of the method is still a boolean instead of `0` / `1`.

Even if it should not change the outcome of current functional code, it's better to keep the same return type.